src/constants/dither/Dither.ts

```diff
@@ -0,0 +1,20 @@
+export const THRESHOLD_1X1 = 0;
```

Comment:
Need to add Dither.ts to index

Reply:
As discussed, for now we won't export it in index.ts